deploy/grafana/livekit-server-overview.json

```diff
@@ -598,7 +598,7 @@
         },
         {
           "exemplar": true,
-          "expr": "sum(rate(livekit_node_service_operation{status=\"success\", type=\"signal_ws\"}[5m])) - sum(rate(livekit_node_service_operation{status=\"success\", type=\"ice_connection\"}[5m]))",
+          "expr": "sum(rate(livekit_node_service_operation{status=\"success\", type=\"signal_ws\"}[5m])) - (sum(rate(livekit_node_service_operation{status=\"success\", type=\"ice_connection\"}[5m]))/2)",
```

Comment:
We create a publisher and subscriber per participant in transportmanager so there are 2 ice_connection to 1 signal_ws